ElectronClient/gui/NoteList/NoteList.tsx

```diff
@@ -141,7 +141,7 @@ class NoteListComponent extends React.Component {
 	}
 
 	dragTargetNoteIndex_(event:any) {
-		return Math.abs(Math.round((event.clientY - this.itemListRef.current.offsetTop()) / this.itemHeight)) - 1;
+		return Math.abs(Math.round((event.clientY - this.itemListRef.current.offsetTop() + this.itemListRef.current.offsetScroll()) / this.itemHeight));
```

Comment:
Was the - 1 at the end of the dragging notes calculation on purpose?
It does not line up with the mouse if it is there.
It makes the drag location 1 note high.

Reply:
Yes I've added that as the index somehow was off by 1 at some point while I was testing. But maybe it was some other issue, or your change fixes it properly. I'll give it a try to be sure.

Reply:
OK, I'm not sure which way it needs to be.

Reply:
How's this look?

Reply:
I will test this change once I get back to the UI update, which should be relatively soon.